framework/vulkan_sample.h

```diff
@@ -99,244 +101,274 @@ namespace vkb
  * - Core classes: Classes in vkb::core wrap Vulkan objects for indexing and hashing.
  */
 
+template <bool CppBindings>
 class VulkanSample : public Application
 {
+	/// <summary>
+	/// PUBLIC INTERFACE
+	/// </summary>

```

Comment:
```suggestion
	/// </summary>
	template <typename CppType, typename CType>
	conditional_type : public std::conditional<CppType, CType, CType>{};

	using RenderContext = conditional_type<vkb::rendering::HPPRenderContext, vkb::RenderContext>::type;
```

Reply:
Good point! Makes the conditional code less verbose.

Reply:
Done.